api/tests/test_hyp3_api.py

```diff
@@ -23,10 +23,17 @@ def batch_stub():
         stubber.assert_no_pending_responses()
 
 
-def submit_job(client, granule, batch_stub=None):
+def submit_job(client, granule, batch_stub=None, email='john.doe@example.com'):
     if batch_stub:
         add_response(batch_stub, granule)
-    return client.post(JOBS_URI, json={'granule': granule})
+    payload = {
+        'process_type': 'RTC_GAMMA',
+        'email': email,
+        'parameters': {
+            'granule': granule
+        }
+    }
+    return client.post(JOBS_URI, json=payload)
 
 
```

Comment:
missing email, invalid email, missing process type, invalid process type, on the fence about testing these but I think once I've bothered to write tests I'm almost always happy to have them